### PR TITLE
[14.0][MIG] stock_production_lot_active: Migration to 14.0

### PR DESCRIPTION
**Pull request by janikvonrotz**
Standard migration from 13.0 to 14.0.

Note: This is my first migration. I followed this guide: <https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration>

Nothing had to be updated to make it work in Odoo 14.0.

My open points are:
- Update README.rst automatically
- Adding myself to contribution
- Define and run tests?
- Contact with the maintainer @ThomasBinsfeld 

Any help on what I a missing here is welcome :+1:

### CODE REVIEW

**Comment by janikvonrotz** [comment]
I get these issues from the pre-commit check:

```
 prettier + plugin-xml....................................................Failed
- hook id: prettier
- files were modified by this hook

stock_production_lot_active/views/stock_production_lot.xml

eslint...............................................(no files to check)Skipped
- hook id: eslint
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing stock_production_lot_active/readme/CONTRIBUTORS.rst
```
Not sure how to resolve them.

**Comment by janikvonrotz** [comment]
@OCA-git-bot merge

**Comment by janikvonrotz** [comment]
@ThomasBinsfeld Are you still the maintainer of this module?

**Review by Abranes — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Thank you for your contribution! Can you please squash the lint fix into the migration commit? I see you added yourself as an author, but in the case of a 'simple' migration like this one, adding yourself as a contributor (which you also did) is more appropriate.

**Comment by janikvonrotz** [comment]
@StefanRijnhart Thanks for the feedback. I have updated the manifest and squashed the commits.

File: stock_production_lot_active/views/stock_production_lot.xml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright 2020 ACSONE SA/NV
-     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```
Comment (StefanRijnhart) [inline stock_production_lot_active/views/stock_production_lot.xml]: Just curious, did pre-commit remove the copyright statement, or did you do that? I don't know if there ever was any agreement in the community about copyright statement on data files, but I would not remove them manually.

Comment (janikvonrotz) [reply stock_production_lot_active/views/stock_production_lot.xml]: Good question, can't remember. Should I restore the copyright statement?

Comment (StefanRijnhart) [reply stock_production_lot_active/views/stock_production_lot.xml]: Yes, please give it a try. If it disappears again we'll know it's the pre-commit doings!

File: default.vim [hunk 2]
```diff
@@ -0,0 +1,85 @@
+" ~/Odoo-Development/oca/stock-logistics-workflow/default.vim:
```
Comment (sebalix) [inline default.vim]: This file has to be removed :)

**Review by sebalix — COMMENTED** [review]
One comment, LG

File: stock_production_lot_active/models/stock_production_lot.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-# Copyright 2020 ACSONE SA/NV
```
Comment (StefanRijnhart) [inline stock_production_lot_active/models/stock_production_lot.py]: Copyrights on Python files surely should not be deleted.

File: default.vim (hunk 2 above)
Comment (sebalix) [reply default.vim]: The file is still there

**Review by sebalix — CHANGES_REQUESTED** [review]

File: stock_production_lot_active/models/stock_production_lot.py (hunk 3 above)
Comment (sebalix) [reply stock_production_lot_active/models/stock_production_lot.py]: @janikvonrotz better to keep the file untouched to reduce the diff to the bare minimum, the copyright was on the first line

**Review by sebalix — CHANGES_REQUESTED** [review]

File: default.vim (hunk 2 above)
Comment (janikvonrotz) [reply default.vim]: Mhm, for some reason deleting the file and commiting with --amend option does not work. Will remove it and then squash the commit.

Comment (sebalix) [reply default.vim]: @janikvonrotz still not ;) Use `git rm --cached default.vim` to remove the file from the git repo without removing it from your filesystem if it is what you want.

Comment (janikvonrotz) [reply default.vim]: @sebalix Using vim to run git rebase interactively seems to be the issue...

**Review by sebalix — APPROVED** [review]
@janikvonrotz thanks! LG

**Comment by janikvonrotz** [comment]
@sebalix Thanks for the patience :)

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**